client/js/analysis.js

```diff
@@ -130,9 +130,9 @@ function determineOverall(json, time) {
     isClear = 'CLEARLY ';
   }
 
-  if (score < -0.2) {
+  if (score < -0.35) {
     tone = 'NEGATIVE';
-  } else if (score > 0.2) {
+  } else if (score > 0.35) {
```

Comment:
how are you getting these bounds?

Reply:
https://cloud.google.com/natural-language/docs/basics#interpreting_sentiment_analysis_values

Reply:
Please add that in a comment, I was wondering too